### PR TITLE
Update transform_age utility

### LINKED ISSUE #86: add age transform handling for formats that are already correct
If I have a `.tsv` where age is already in the right format (e.g. `float` or `int`), then our CLI currently doesn't know what to do with it (because `float` or `int` are not recongnized heuristics. 

There are quite a lot of datasets where age is encoded like that, so this is a problem that needs fixing.

### LINKED ISSUE #87: age transform should only handle string values

### PR DESCRIPTION
**Pull request by alyssadai**
Closes #86, closes #87

### CODE REVIEW

File: bagelbids/pheno_utils.py [hunk 1]
```diff
@@ -113,7 +113,9 @@ def get_age_heuristic(column: str, data_dict: dict) -> str:
     return data_dict[column]["Annotations"]["Transformation"]["TermURL"]
 
 
-def transform_age(value: Union[int, float, str], heuristic: str) -> float:
+def transform_age(value: str, heuristic: str) -> float:
+    if heuristic in ["bg:float", "bg:int"]:
```
Comment (surchs) [inline bagelbids/pheno_utils.py:117]: this is great, but we should probably also cast the input `value` to str where we're calling `transform_age`, e.g. [here](https://github.com/neurobagel/bagel-cli/pull/92/files#diff-b0983ec8b7729431df0779f42e11ff918eacd934e91a6df1fde5a92c28ba1698R155)

File: bagelbids/tests/test_utility.py [hunk 2]
```diff
@@ -112,6 +112,8 @@ def test_get_assessment_tool_availability(
 @pytest.mark.parametrize(
     "raw_age,expected_age,heuristic",
     [
+        ("11.0", 11.0, "bg:float"),
+        ("11", 11.0, "bg:int"),
```
Comment (surchs) [inline bagelbids/tests/test_utility.py:116]: :tada:

**Review by surchs — APPROVED** [review]
Awesome, thanks @alyssadai for fixing this so quickly. Looks great, good to go! :cook:

File: bagelbids/pheno_utils.py (hunk 1 above)
Comment (alyssadai) [reply bagelbids/pheno_utils.py:117]: good catch

**Pull request merged**